lib/ups/builders/builder_base.rb

```diff
@@ -218,10 +230,11 @@ def shipment_service_options
         end
       end
 
-      def packaging_type
-        code_description 'PackagingType', '02', 'Customer Supplied'
+      def packaging_type(packaging_type)
```

Comment:
`packaging_type` not really descriptive, it's not the type but rather the hash format